CMakeLists.txt

```diff
@@ -1,43 +1,62 @@
-cmake_minimum_required (VERSION 3.2.2)
+cmake_minimum_required (VERSION 3.4.3)
+project(sycl-blas VERSION 0.1)
 
-project(sycl-blas)
+include(${CMAKE_BINARY_DIR}/conan_paths.cmake OPTIONAL)
+include(CTest)
 
-enable_testing()
+list(APPEND CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake/Modules")
+find_package(OpenCL REQUIRED)
+find_package(ComputeCpp REQUIRED)
+find_package(OpenBLAS REQUIRED)
```

Comment:
I'm not sure that this is the behavior we want here - although we should always have access to OpenBLAS, we might want to use a different BLAS library for tests and thus should let the user configure it.

Reply:
@AdamHarries yes the OpenBLAS should not be mandatory. The default should be OpenBLAS but if user provide  any BLAS library, the user BLAS library should be used instead.

Reply:
I removed OpenBLAS from the list of required libraries and added a search for other BLAS implementations, although the latter part is untested for now.

A 3rd party BLAS will only be required for the tests and won't be exported as a dependency.